substrate-airgapped/examples/signed_tx_from_pair.rs

```diff
@@ -1,49 +1,46 @@
 use codec::Encode;
-use sp_core::H256;
+use primitive_types::H256;
 use sp_runtime::{generic::Header, traits::BlakeTwo256, DeserializeOwned};
 use substrate_airgapped::{
 	balances::Transfer, CallIndex, GenericCall, KusamaRuntime, MortalConfig, Mortality, Tx,
 	TxConfig,
 };
 
 // Example deps
+use anyhow::anyhow;
 use hex;
 use reqwest;
 use serde::{Deserialize, Serialize};
 use sp_keyring::AccountKeyring;
 use sp_version::RuntimeVersion;
+use std::str::FromStr;
 
-fn main() -> Result<(), Box<dyn std::error::Error>> {
+type KusamaTransfer = Transfer<KusamaRuntime>;
+type Error = anyhow::Error;
+
+fn main() -> Result<(), Error> {
 	// Get the latest block hash and then make all non historic queries at that block.
-	let block_hash = rpc_to_local_node::<(), String>("chain_getBlockHash", vec![])?.result;
-	let runtime_version = rpc_to_local_node::<String, RuntimeVersion>(
-		"chain_getRuntimeVersion",
-		vec![block_hash.clone()],
-	)?
-	.result;
-	let header = rpc_to_local_node::<String, Header<u32, BlakeTwo256>>(
-		"chain_getHeader",
-		vec![block_hash.clone()],
-	)?
-	.result;
-	let genesis_hash = rpc_to_local_node::<usize, String>("chain_getBlockHash", vec![0])
-		.and_then(|rpc_res| Ok(string_to_h256(&rpc_res.result)))?;
-	let block_hash = string_to_h256(&block_hash[..]);
-
-	type TransferType = Transfer<KusamaRuntime>;
+	let block_hash: H256 = rpc_to_local_node("chain_getBlockHash", Vec::<()>::new())?.result;
+	let runtime_version: RuntimeVersion =
+		rpc_to_local_node("chain_getRuntimeVersion", vec![block_hash.clone()])?.result;
```

Comment:
H256 implements `Copy`

```suggestion
		rpc_to_local_node("chain_getRuntimeVersion", vec![block_hash])?.result;
```